lib/matplotlib/projections/geo.py

```diff
@@ -392,17 +394,18 @@ def transform_non_affine(self, xy):
             x, y = xy.T
             # from Equations (7, 8) of
             # https://mathworld.wolfram.com/MollweideProjection.html
-            theta = np.arcsin(y / np.sqrt(2))
-            longitude = (np.pi / (2 * np.sqrt(2))) * x / np.cos(theta)
-            latitude = np.arcsin((2 * theta + np.sin(2 * theta)) / np.pi)
+            sqrt2 = 2 ** (1 / 2)
+            theta = np.arcsin(y / sqrt2)
+            longitude = (math.pi / (2.0 * sqrt2)) * x / np.cos(theta)
```

Comment:
again sqrt2 doesn't warrant being a separate variable; the compiler will inline `2**(1/2)` in theta and `2**(3/2)` in longitude; and again `2.0` -> `2`